native/sapp-wasm/src/gl.rs

```diff
@@ -739,8 +739,8 @@ extern "C" {
 extern "C" {
     pub fn glClearStencil(s: GLint);
 }
-extern "C" {
-    pub fn glColorMask(red: GLboolean, green: GLboolean, blue: GLboolean, alpha: GLboolean);
+pub fn glColorMask(red: GLboolean, green: GLboolean, blue: GLboolean, alpha: GLboolean) {
+    // TODO: marking this extern "C" does not work on WASM,  shows: Uncaught (in promise) LinkError: WebAssembly.instantiate(): Import #46 module="env" function="glColorMask" error: function import requires a callable
```

Comment:
I am looking on colorMask and stencil function in docs and it looks that they should be supported in webgl1: 

https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/colorMask
https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/stencilOp

The other clue that they are probably supported: there is no special code in orignal emscripten js code: https://github.com/emscripten-core/emscripten/blob/master/src/library_webgl.js (just search for appropriate function, its a long file)

So my guess is that if you add all that function to gl.js - they will work in webgl. 

Something like this will fix wasm runtime errors and I guess will work on webgl. But hard to say without tests, I am never sure when its about web
```
--- a/native/sapp-wasm/js/gl.js
+++ b/native/sapp-wasm/js/gl.js
@@ -431,6 +431,9 @@ var importObject = {
         glClearStencil: function (s) {
             gl.clearColorStencil(s);
         },
+        glColorMask: function (red, green, blue, alpha) {
+            gl.colorMask(red, green, blue, alpha);
+        },
         glScissor: function (x, y, w, h) {
             gl.scissor(x, y, w, h);
         },
```

Reply:
Thanks - will try the proposed fix!